packages/theme/src/default-theme/foundational-styles/colors.ts

```diff
@@ -4,6 +4,7 @@ import fills from './fills'
 
 const colors = {
   background: {
+    default: 'white',
```

Comment:
Put this inside:

const colors = {
  background: {...},
  labelWrapper: {
    background: 'white'
  }
}
```

Reply:
@hzub But then the background of a parent would have to get its color from labelWrapper property instead of background.default.

Reply:
@Idered well, @Nelias is right - I guess we want developers to use one standarized background in context?

Reply:
I changed it back to one default background for both.